raster-foundry/raster-foundry
# Remove caching for RasterSource and Tiles

## Overview
Remove references to tile and raster source cache configuration

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

- re-assemble your api server and backsplash-server jars. Verify that there are no compiler errors
- Run `./scripts/server` and verify that there are no runtime errors when rendering tiles

Closes #5177
